packages/clarity-js/src/layout/mutation.ts

```diff
@@ -169,19 +170,22 @@ function track(m: MutationRecord, timer: Timer): string {
     // In those cases, IDs will change however the selector (which is relative to DOM xPath) remains the same
     let key = [parent, element, m.attributeName, names(m.addedNodes), names(m.removedNodes)].join();
     // Initialize an entry if it doesn't already exist
-    history[key] = key in history ? history[key] : [0];
+    history[key] = key in history ? history[key] : [0, instance];
```

Comment:
Minor; Should  this really be three thing as h[2] stores list of nodes.

Reply:
As defined in the type... the last parameter is optional. So we don't have to explicitly give a null value. From the layout.d.ts -> export interface MutationHistory {
    [key: string]: [/* Count */ number, /* Instance */ number, /* Remove Nodes Buffer */ NodeList?];
}